mappings/net/minecraft/container/DispenserContainer.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS sh net/minecraft/container/DispenserContainer
+	FIELD a entity Laz;
```

Comment:
why is there an entity

Reply:
because of good valo names that DispenserBlockEntity is DispenserEntity

Reply:
nice, just apply that name suggestion above 
need this merged!

Reply:
`Good valo names`...
they're called cursed names excuzme